docs/kyma/10-04-common-errors.md

```diff
@@ -56,3 +56,16 @@ helm history {RELEASE_NAME} --tls
 ```
 
 >**NOTE:** Names of Helm releases correspond to names of Kyma components.
+
+## The server could not find the requested resource
+
+During the installation process, you may encounter the `server could not find the requested resource` error that reports a misspelled CRD name:
+```
+Details: Helm install error: rpc error: code = Unknown desc = release compass failed: the server could not find the requested resource (post gatewaies.networking.istio.io)
+```
+Tiller in older versions prepares plural names using a set of rules, instead of reading them from the CRD. This method not always produces the proper word. For example, `gatewaies` instead of `gateways`.
```

Comment:
```suggestion
Tiller in older versions prepares names plurals using a set of rules, instead of reading them from the CRD. This method does not always produce the correct plural form. For example, for `gateway` it produces `gatewaies` instead of `gateways`.
```